pandas/core/aggregation.py

```diff
@@ -40,7 +43,9 @@ def is_multi_agg_with_relabel(**kwargs) -> bool:
     )
 
 
-def normalize_keyword_aggregation(kwargs: dict) -> Tuple[dict, List[str], List[int]]:
+def normalize_keyword_aggregation(
+    kwargs: dict,
```

Comment:
this should be Dict from typing. can you see if you can also add type parameters to Dict.